content/docs/introduction/getting_started.md

```diff
@@ -86,6 +86,10 @@ Blindly setting `GOMAXPROCS` to a high value can be
 counterproductive. See the relevant [Go
 FAQs](http://golang.org/doc/faq#Why_no_multi_CPU).
 
+Notice that Prometheus by default uses around 3GB in memory, if you have smaller
+machine, you can change memory usage configuration to reduce it. See details in
```

Comment:
Missing article for "memory usage configuration", but I guess I'd just say "you can tune Prometheus to use less memory."